### PR TITLE
msc: cleanup mixins

### PR DESCRIPTION
**Pull request by Wyvest**
## Checklist
- [ ] I made a clear description of what was changed
- [ ] I stated why these changes were necessary
- [ ] I updated documentation or said what needs to be updated
- [ ] I made sure these changes are backwards compatible
- [ ] This pull request is for one feature/bug fix

### CODE REVIEW

File: versions/1.16.5-forge/src/main/java/cc/polyfrost/oneconfig/internal/mixin/KeyboardMixin.java [hunk 1]
```diff
@@ -36,8 +36,15 @@
...
-    @Inject(method = "onKeyEvent", at = @At(value = "INVOKE", target = "Lnet/minecraftforge/client/ForgeHooksClient;fireKeyInput(IIII)V", remap = false), remap = true)
+    @Inject(method = "onKeyEvent", at = @At(
+            //#if FORGE
+            value = "INVOKE", target = "Lnet/minecraftforge/client/ForgeHooksClient;fireKeyInput(IIII)V", remap = false
+            //#else
+            //$$ "TAIL"
+            //#endif
+    ), remap = true)
     private void onKeyEvent(long windowPointer, int key, int scanCode, int action, int modifiers, CallbackInfo ci) {
+        System.out.println("gdgjasdkogdsagikoDSJMGdskogjmds");
```
Comment (xtrm-en) [inline versions/1.16.5-forge/src/main/java/cc/polyfrost/oneconfig/internal/mixin/KeyboardMixin.java:47]: damn

**Review by xtrm-en — CHANGES_REQUESTED** [review]

File: versions/1.16.5-forge/src/main/java/cc/polyfrost/oneconfig/internal/mixin/KeyboardMixin.java (hunk 1 above)
Comment (pauliesnug) [reply versions/1.16.5-forge/src/main/java/cc/polyfrost/oneconfig/internal/mixin/KeyboardMixin.java:47]: samesies

Comment (Rattlyy) [reply versions/1.16.5-forge/src/main/java/cc/polyfrost/oneconfig/internal/mixin/KeyboardMixin.java:47]: real

**Review by xtrm-en — DISMISSED** [review]
yea

**Review by xtrm-en — APPROVED** [review]
lgtm

**Review by pauliesnug — APPROVED** [review]
llll (looks looks looks looks)

**Pull request merged**